README.md

```diff
@@ -12,9 +12,22 @@ DOCUMENT_DATABASE_PORT=5432
 DOCUMENT_DATABASE_PASSWORD=xxxx
 ```
 
+### Running a local Development Database
```

Comment:
@jimmylee lmk how you've been testing locally. I made it super easy to run a local `postgres` in docker here for testing new stuff.